code/14-location-figures.R

```diff
@@ -57,38 +54,19 @@ input_tidy = dplyr::mutate(
 input_ras = terra::rast(input_tidy, type = "xyz", crs = "EPSG:3035")
 
 # reproject German outline
-ger = terra::project(ger, crs(input_ras))
+ger = st_as_sf(terra::project(ger, crs(input_ras)))
 
 # 2.2 Create figure========================================
 #**********************************************************
+tm_1 = tm_shape(input_ras) +
```

Comment:
And this does the same as the `spplot` code?? This is impressive, and I should definitely look more into **tmap**.

Reply:
I tried to made new figures as similar to the old ones as possible (the legend position is the biggest difference).

Reply:
excellent work, Jakub, thanks a lot!